frontend/src/views/PrinterControlPage.vue

```diff
@@ -304,6 +315,42 @@ export default {
   },
 
   methods: {
+    calculateAtLeastOneFeedPortrait() {
```

Comment:
Can you make it so that it uses watcher? Calling this method directly in the code is more bug-prone in case when we add more code to handle webcam orientation and forget to call this.